starter-code/apple-pie-starter-code/styles.css

```diff
@@ -0,0 +1,64 @@
+/* styles.css */
+body {
+  font-family: 'Open Sans', sans-serif;
+}
+
+header {
+  background-image: url('images/apple-pie.jpg');
+  block-size: 550px;
+}
+
+h1 {
+  padding: 250px;
+  color: white;
+  text-align: center;
+  font-size: 3.2em;
+}
+
+.description {
+  color: gray;
+  font-size: 1.5em;
+}
+.block {
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+  width: 60%;
+  padding-top: 30px;
+  padding-bottom: 30px;
+} 
+.recipe-info {
+  display: block;
+  margin-left: auto;
+  margin-right: auto; 
+  padding: 0;
+}
+
+#Ingredients li {
+  list-style-type: none;
+}
+
+
+.text {
```

Comment:
Instead calling text class on each `li` element in your .html file, you can do here: 
```
.block ul li {
	// styles go here
}
```